Apps/sonar-config.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  This file defines properties which would be understood by the SonarQube Scanner for MSBuild, if not overridden (see below)
```

Comment:
What if we want to run the analysis on a single project instead of the solution?

Reply:
Both running for the solution or on single project uses the same configuration file, just provide the path when running the /begin command